htdocs/takepos/invoice.php

```diff
@@ -1014,7 +1014,7 @@ function fail($message)
 // Set focus to search field
 if ($action == "search" || $action == "valid") {
 	?>
-	parent.ClearSearch(true);
+	parent.ClearSearch(false);
```

Comment:
@altairis-melina why did you change ?

Reply:
@altairis-melina in fact, eldy asked you to add false parameter in the call of other ClearSearch, nut this one is ok